uber/config.py

```diff
@@ -303,6 +303,28 @@ def REMAINING_BADGES(self):
     def ADMIN_ACCESS_SET(self):
         return sa.AdminAccount.access_set()
 
+    @request_cached_property
+    def EMAIL_APPROVED_IDENTS(self):
+        return self.get_approved_idents()
+
+    # TODO: only reason this is a separate method is because for unit tests we patch this method,
+    # and monkeypatch.setattr() doesn't like to patch EMAIL_APPROVED_IDENTS
+    # when it's decorated with @request_cached_property
+    def get_approved_idents(self):
+        with sa.Session() as session:
+            return {ae.ident for ae in session.query(sa.ApprovedEmail)}
+
```

Comment:
@EliAndrewC could use a hand with one thing. I moved these so we can do c.EMAIL_APPROVED_IDENTS (a list of idents approved to send) and they're cached with request_cached_property (which the daemon manages the lifetime of), but I can't get my unit tests to override c.EMAIL_APPROVED_IDENTS with fake data.

I resorted to having c.EMAIL_APPROVED_IDENTS which calls ```get_approved_idents``` and it's THAT method I'm overridding.  I was wondering if you knew offhand what I have to do to monkeypatch c.EMAIL_APPROVED_IDENTS directly.

I was doing this:
```
monkeypatch.setattr(c, 'EMAIL_APPROVED_IDENTS', Mock(return_value={...stuff...}))
```

but it was coming back saying 'no attribute found' and I suspect it's the wrapping going on with the request_cached_property decorator.  any ideas?